project/app/Helpers/ToHTMLstorage.php

```diff
@@ -25,8 +25,18 @@ public function itemCollectionToHTMLst(Collection $items): string
             </tr>";
     }
 
-    public function itemToHTML(Item $item): string
+    public function itemToHTML(Item $item, bool $add_form): string
```

Comment:
Nie myślałem, że aż tak głęboko trzeba będzie do tego w chodzić... Dobrze, że udało się to ogarnąć

Reply:
Spoko